src/metldata/builtin_transformations/copy_content/assumptions.py

```diff
@@ -39,4 +39,4 @@ def check_model_assumptions(
             assert_class_is_source(path=path, instruction=instruction)
             assert_path_classes_and_relations_exist(model=schema, path=path)
             assert_no_relation_target_multiplicity(model=schema, path=path)
-            assert_object_path_exists(model=schema, instruction=instruction)
+            assert_object_path_exists(model=schema, instruction=instruction)  # type: ignore
```

Comment:
Please see my comment below about using `NewContentSchemaPath` in `copy_content`.